index.html

```diff
@@ -0,0 +1,380 @@
+        </div>
+        <div>
+          <img
+            class="images"
+            src="./assets/project_img4.png"
+            alt="project_img1"
+          />
+        </div>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="container">
+        <div class="row">
+          <div class="column">
+            <div class="about-description">
+              <h3 class="about-introduction">
+                About <br />
+                Myself
+              </h3>
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+              <h4>LET'S CONNECT</h4>
+              <div class="social-media">
+                <ul>
+                  <li>
+                    <a href="https://twitter.com/M_ayienga"></a
+                    ><i class="fab fa-github"></i>
+                  </li>
+                  <li>
+                    <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+                      ><i class="fab fa-linkedin"></i
+                    ></a>
+                  </li>
+                  <li>
+                    <a href="https://www.instagram.com/merl.__/?hl=en"
+                      ><i class="fab fa-angellist"></i
+                    ></a>
+                  </li>
+                  <li>
+                    <a href="https://twitter.com/M_ayienga"
+                      ><i class="fab fa-twitter"></i>
+                    </a>
+                  </li>
+                  <li>
+                    <a href="marlymayienga@gmail.com"
+                      ><i class="fab fa-medium-m"></i>
+                    </a>
+                  </li>
+                </ul>
+              </div>
+              <div>
+                <button class="resume">Get My Resume</button>
+              </div>
+            </div>
+          </div>
+          <div class="column">
+            <div class="container-technologies">
+              <div class="languages">
+                <p class="languages-title">
+                  Languages <i class="fas fa-angle-down"></i>
+                </p>
+                <div class="web-languages">
+                  <div class="web-languages-card">
+                    <img src="./assets/js.png" alt="Javascript" />
+                    <div class="container">Javascript</div>
+                  </div>
+                  <div class="web-languages-card">
+                    <img src="./assets/html.png" alt="html" />
+                    <div class="container">HTML</div>
+                  </div>
+                  <div class="web-languages-card">
+                    <img src="./assets/css.png" alt="css" />
+                    <div class="container">CSS</div>
+                  </div>
+                </div>
```

Comment:
- [ ] [OPTIONAL] Please change the `div` tag to `li` tags and put the whole list into a `ul` tag. The technology categories should be in a `ul` tag and each of the technologies in a `li` tag.

Reply:
Hello @wuletawwonte, it would be easier to use a list for this section if there were no hr lines in between them. It is not possible to use a list in between the hr because it goes against the HTML linter rules. 
Kindly advise if there is another way to go about the issue